public/app/plugins/panel/trend/TrendPanel.tsx

```diff
@@ -137,15 +136,6 @@ export const TrendPanel = ({
                 timeZone={timeZone}
               />
             )}
-
-            <ContextMenuPlugin
-              data={alignedDataFrame}
-              frames={info.frames!}
-              config={config}
-              timeZone={timeZone}
-              replaceVariables={replaceVariables}
-              defaultItems={[]}
-            />
```

Comment:
probably does not make sense in this panel, since x != time. (most likely a leftover from TimeSeries copy/paste)